code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1920,6 +1920,16 @@
 	reagent_data = list(/decl/reagent/nutriment = list("bread" = 3, "cheese" = 3))
 	bitesize = 2
 
+/obj/item/reagent_containers/food/snacks/reubensandwich
+	name = "reuben sandwich"
+	desc = "A toasted sandwich packed with savory, meat and sour goodness!"
+	icon_state = "reubensandwich"
+	filling_color = "BF8E60"
```

Comment:
```suggestion
	filling_color = "#BF8E60"
```

You could use color defines for all of these to make things easier. But probably none of the other hundreds of recipes do so... 🤷 

Check out `code/__defines/color.dm` for those.

Reply:
Unfortunately the ``filling_color`` is used for the custom recipes (sandwiches/breads/jelly/cereal/jawbreaker/sucker/candybars/donks/donuts/pies/kabob/waffles) to help distinguish what exactly a thing is made out of, so people tend to go with their own custom color choices.

It's hell